src/pages/LoginPage.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { FC } from "react";
+import { NavLink, useNavigate } from "react-router-dom";
+import { Button, Form, Container, Card } from "react-bootstrap";
+import { useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import * as Yup from "yup";
+import ky from "ky";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import "../styles/Auth.scss";
+import { useDispatch } from "react-redux";
+import { loginUser } from "../store/actionCreators";
+
+type UserLoginForm = {
+  email: string;
+  password: string;
+};
+
+type ResponseLogin = {
+  token: string;
+  message: string;
+};
+
+const LoginPage: FC = () => {
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+  const validationSchemaLogin = Yup.object().shape({
+    email: Yup.string().required("Email обязателен").email("Неверный email"),
+    password: Yup.string()
+      .required("Пароль обязателен")
+      .min(6, "Пароль должен состьять из 6 символов")
+      .max(20, "Пароль не может иметь больше 20 сиволов"),
+  });
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<UserLoginForm>({
+    resolver: yupResolver(validationSchemaLogin),
+  });
+  const onSubmitRegistration: any = async (data: UserLoginForm) => {
+    try {
+      const response: ResponseLogin = await ky
+        .post("http://localhost:5000/api/users/login", {
+          json: { email: data.email, password: data.password },
+        })
+        .json();
+      navigate("/mypage");
+      toast.success(response.message);
+      dispatch(loginUser(true, response.token));
+    } catch (error: any) {
+      if (error.name === "HTTPError") {
+        const errorJson = await error.response.json();
+        toast.error(errorJson.message);
+      }
+    }
+  };
+  return (
+    <div>
+      <ToastContainer position="top-center" autoClose={5000} theme="colored" />
+      <Container className="d-flex justify-content-center align-items-center ">
+        <Card className="p-5 m-5 FormLogin">
+          <h2 className="m-auto">Авторизация</h2>
+          <Form
+            className="d-flex flex-column mt-3"
+            onSubmit={handleSubmit(onSubmitRegistration)}
+          >
+            <Form.Label>Email</Form.Label>
+            <Form.Control
+              type="email"
+              {...register("email")}
+              className={`form-control ${errors.email ? "is-invalid" : ""}`}
+            />
+            <div className="invalid-feedback">{errors.email?.message}</div>
+            <Form.Label>Пароль</Form.Label>
+            <Form.Control
+              type="password"
+              {...register("password")}
+              className={`form-control ${errors.password ? "is-invalid" : ""}`}
```

Comment:
https://www.npmjs.com/package/classnames

This package can bring to the project more readability